suite-common/wallet-utils/src/reviewTransactionUtils.ts

```diff
@@ -181,7 +184,7 @@ const constructOldFlow = ({
                 value: precomposedForm.rippleDestinationTag,
             });
         }
-    } else if (!precomposedTx.useNativeRbf) {
+    } else if (isRbf && !precomposedTx.useNativeRbf) {
```

Comment:
I have to admit I'm totally missing a bigger picture here, but this is different than before, right? This branch used to apply for non-native rbf txs **and** non-rbf txs, now it completely ignores non-rbf txs. Is it wanted?

Reply:
It is there to satisfy TypeScript. `useNativeRbf` does not exist on `PrecomposedTransactionCardanoFinal` so I have to use `isRbfTransaction` as a type guard.

Reply:
I know, I'm talking about `precomposedTx.useNativeRbf === undefined` case, which satisfied the condition before but not now. Is it something which can happen?

Reply:
6774c3f5c385da7dd29e8140f6d4129de04ed7da